Userland/Libraries/LibGUI/GML/Formatter.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021, Linus Groh <linusg@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Forward.h>
+#include <AK/String.h>
+#include <LibGUI/GML/AST.h>
+#include <LibGUI/GML/Parser.h>
+
+namespace GUI::GML {
+
+String format_gml(StringView);
+
+String format_gml(StringView string)
```

Comment:
this declaration then definition looks a bit odd. doesn't the function need to be `inline` in case multiple translation units include this file?

Reply:
If I remove the declaration, Clang complains that there isn't a declaration. If I make it static, it gets duplicated across translation units (I think).

Reply:
```cpp
inline String format_gml(StringView string)
```